src/scalems/messages.py

```diff
@@ -254,14 +263,19 @@ class AddItem(Command[str]):
 
     key: typing.ClassVar = "add_item"
 
-    def __init__(self, encoded_item: str):
-        # TODO: This should be a public scalems schema, but is currently assumed
-        #  to be scalems.radical.raptor.ScalemsRaptorWorkItem.
-        self._encoded_item = encoded_item
+    # TODO: This should be a public scalems schema, but is currently assumed
+    #  to be a JSON-serialized scalems.radical.raptor.ScalemsRaptorWorkItem.
+    _encoded_item: str
+
+    # def __init__(self):
+    #     # With a Protocol parent, we need an __init__ to establish this as a concrete class.
+    #     ...
```

Comment:
I can't remember where this came from. In at least one version of Python, there was some sort of internal metaprogramming (maybe `isinstance()` or `functools.singledispatch`? or maybe a check related to ABC or metaclass hierarchy) that spit an error at some point when a concrete class did not find a `__init__()` before finding a `typing.Protocol` when traversing the MRO. It doesn't seem to be causing a problem here and now.

If we encounter the problem again, we should document it better.
```suggestion
```